src/tests/Mission.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Mission } from '../../__mocks__/components/MissionMock';
+
+describe('Mission', () => {
+  const component = (
+    <Mission
+      id="1"
+      mission="Moon"
+      description="the moon"
+    />
+  );
+
+  test('should match snapshot', () => {
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
- [ ] I understand this may be stressful to consider applying a few more unit tests for the mission. However, as the previous review suggests, consider adding a few more unit tests to ensure that the functionality of the application is working as intended ☺️.